l1-contracts/foundry.toml

```diff
@@ -42,6 +42,7 @@ remappings = [
 ]
 optimizer = true
 optimizer_runs = 200
+allow_internal_expect_revert = true
```

Comment:
same as in the previous pr, we should not allow this thing globally

Reply:
Yes, removed.